velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -662,4 +662,37 @@ TEST_F(BinaryFunctionsTest, toIEEE754Bits32) {
       hexToDec("FF7FFFFF"),
       toIEEE754Bits32(std::numeric_limits<float>::lowest()));
 }
+
+TEST_F(BinaryFunctionsTest, rpad) {
+  const auto rpad = [&](std::optional<std::string> arg,
+                        std::optional<int32_t> size,
+                        std::optional<std::string> key) {
+    return evaluateOnce<std::string>("rpad(c0, c1, c2)", arg, size, key);
+  };
+  EXPECT_EQ(rpad("1234", 14, "45"), ("12344545454545"));
+  EXPECT_EQ(rpad("1234", 14, "4524"), ("12344524452445"));
+  EXPECT_EQ(rpad("1234", 6, "4524"), ("123445"));
+  EXPECT_EQ(rpad("23", 0, "4524"), (""));
```

Comment:
Please can you add strings with special characters for both input and padding parameters.